FileSystem/Workbook.js

```diff
@@ -0,0 +1,23 @@
+
+var FileSystem = {
+  current_directory: "<ToBeDefined>",
+  api_workbook_open: api_workbook_open_,  
+};
+
+/**
+ * Emulates workbook.Open(filename) API 
+ * Opens the corresponding spreadsheet in a new new tab. 
+ *
+ * @param {string} path The local file path in the Excel file
+ * 
+ * @return {Spreadsheet} The spreadsheet object of the path
+ */
+function api_workbook_open_(path) {
+    var fileId = FileMapper.getFileId(this.current_directory, path, MimeType.GOOGLE_SHEETS);
+    var file = SpreadsheetApp.openById(fileId);
+    openURL(file.getUrl(), `"${file.getName()}" file open in new tab`);
+    return file;
```

Comment:
Do we need to return the file here? I guess openURL does it's job.

Reply:
The return value of Workbook.open returns Workbook. So I thought it is best to replicate that behavior as well. Should I remove it? 

https://docs.microsoft.com/en-us/office/vba/api/excel.workbooks.open#return-value